lib/data/database/tables/groups.dart

```diff
@@ -1,8 +1,10 @@
 import 'package:drift/drift.dart';
 import 'package:trakli/data/database/tables/sync_table.dart';
+import 'package:trakli/data/database/converters/media_converter.dart';
 
 @DataClassName('Group')
 class Groups extends Table with SyncTable {
   TextColumn get name => text()();
   TextColumn get description => text().nullable()();
+  TextColumn get icon => text().nullable().map(const MediaConverter())();
```

Comment:
```suggestion
  TextColumn get icon => text().nullable().map(const MediaConverter())();
```
The relationship between `Icon` and the models is not handled this way in the backend. Can this store files? This might also cause syncing issues.


The recommended way is that you create a new `Icon` model, that stores, `icon_type`, `content`, `iconable_id` and `iconable_type` much like we did for categories?


Then we can link to the models via a relationship.